node_exporter.go

```diff
@@ -82,22 +147,7 @@ func main() {
 	log.Infoln("Starting node_exporter", version.Info())
 	log.Infoln("Build context", version.BuildContext())
 
-	// This instance is only used to check collector creation and logging.
```

Comment:
I don't think you meant to remove this

Reply:
Yes, I did. We don't have a NodeCollector instance anymore that is just there for checking collector creation and logging. Instead, we keep that instance around and use it for unfiltered metrics collection (which should give a bit of a performance boost).

Reply:
This has removed the log of enabled collectors at startup, which was useful

Reply:
No it hasn't. I added a comment to point you to the new place.